pypit/arutils.py

```diff
@@ -400,6 +400,27 @@ def func_val(c, x, func, minv=None, maxv=None):
                    "Please choose from 'polynomial', 'legendre', 'chebyshev', 'bspline'")
 
 
+def calc_fit_rms(xfit, yfit, fit, func, fmin=None, fmax=None):
```

Comment:
For consistency, it might make sense to rename `fmin`, and `fmax` to `minv` and `maxv` (or vice-versa)